### PR TITLE
First try to commit - tests added

### PR DESCRIPTION
**Pull request by kisrefod**
feat: add tests for config file

### CODE REVIEW

**Comment by kisrefod** [comment]
Пробую комментарии в Гитхабе.

File: test/config-test.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import assert from 'assert'
+
+import {host} from './../src/config.js'
+assert.deepStrictEqual(host, 'localhost')
```
Comment (AndreyBelym) [inline test/config-test.js:4]: Please use the typical Mocha test structure (with `describe` and `it`), and use the Chai module for assertions.

**Review by AndreyBelym — COMMENTED** [review]
Good job, keep up! 👍

**Review by AndreyBelym — APPROVED** [review]
Good job! I modified your npm scripts to run lint before tests. Fix linting errors and it is good to merge 🎉

**Comment by AndreyBelym** [comment]
Coverted to draft to prevent accidental merge before fixing lint.

**Comment by AndreyBelym** [comment]
Just a tip: looks like to add `https://www.npmjs.com/package/eslint-plugin-import` to your devDependencies

**Review by AndreyBelym — APPROVED** [review]

**Comment by AndreyBelym** [comment]
Thank you for the excellent job done! Looks good to merge

**Pull request merged**